charts/csi-powermax/values.yaml

```diff
@@ -427,6 +427,20 @@ authorization:
   # Default value: "true"
   skipCertificateValidation: true
 
+# Storage Capacity Tracking
+# Note: Capacity tracking is supported in kubernetes v1.25 and above, this feature will be automatically disabled in older versions.
```

Comment:
@tssushma In this comment, the version should be v1.24 as the feature is supported in k8s v1.24 and above.

Reply:
Changed the version to v1.24